packages/mask/src/components/shared/PersonaListDialog.tsx

```diff
@@ -0,0 +1,330 @@
+import { Icons } from '@masknet/icons'
+import { Avatar, Box, Button, DialogContent, Stack, Typography } from '@mui/material'
+import {
+    BindingProof,
+    EMPTY_LIST,
+    formatPersonaFingerprint,
+    NextIDPlatform,
+    PersonaIdentifier,
+    PersonaInformation,
+    ProfileIdentifier,
+} from '@masknet/shared-base'
+import { useAsync, useAsyncFn, useCopyToClipboard } from 'react-use'
+import { useLastRecognizedIdentity } from '../DataSource/useActivatedUI'
+// TODO: move to share
+import { useConnectedPersonas } from '../DataSource/useConnectedPersonas'
+import Services from '../../extension/service'
+import { InjectedDialog } from '@masknet/shared'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useNextIDVerify } from '../DataSource/useNextIDVerify'
+import { useI18N } from '../../utils'
+import { WalletMessages } from '../../plugins/Wallet/messages'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import { makeStyles, useCustomSnackbar } from '@masknet/theme'
+import { activatedSocialNetworkUI } from '../../social-network'
+
+const useStyles = makeStyles()((theme) => {
+    return {
+        root: {
+            width: 384,
+            height: 386,
+            padding: theme.spacing(1),
+            background: theme.palette.maskColor.bottom,
+        },
+        content: {
+            padding: theme.spacing(0, 2, 2, 2),
+            scrollbarWidth: 'none',
+            '&::-webkit-scrollbar': {
+                display: 'none',
+            },
+        },
+        header: {
+            background: theme.palette.maskColor.bottom,
+        },
+        nickname: {
+            fontSize: 16,
+            lineHeight: '20px',
+            color: theme.palette.maskColor.main,
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            whiteSpace: 'nowrap',
+        },
+        fingerprint: {
+            fontSize: 12,
+            lineHeight: '16px',
+            color: theme.palette.maskColor.second,
+        },
+        copyIcon: {
+            color: 'red',
+        },
+        dot: {
+            display: 'inline-block',
+            background: '#2DDF00',
```

Comment:
Use a theme color.

Reply:
Decided to use use hard code after consult with nuanyang.